sympy/polys/polytools.py

```diff
@@ -61,7 +61,24 @@
 
 @public
 class Poly(Expr):
-    """Generic class for representing polynomial expressions. """
+    """Generic class for representing and operating on polynomial expressions.
+
+        Examples
+        ========
+        >>> from sympy import Poly
+        >>> from sympy.abc import x, y
+        >>> Poly(x*(x**2 + x - 1)**2) # Univariate polynomial
+        Poly(x**5 + 2*x**4 - x**3 - 2*x**2 + x, x, domain='ZZ')
+
+        >>> Poly(x*(x**2 + x - 1)**2, domain = 'R') # Univariate polynomial with specified domain
```

Comment:
Go ahead an put this as text between the above and below `>>>` lines. You can feel free to say more, too, if you want. (You can see many examples of this throughout SymPy.)